### ISSUE TITLE
Feature Request for Review Management Features and Removal of Outdated Reviews for [duplicatefinder]

### ISSUE THREAD

**Issue by eldertek** [nextcloud/appstore#1215]
Hello, 
The reviews on the [DuplicateFinder](https://apps.nextcloud.com/apps/duplicatefinder) app page are outdated and inaccurate. Since I've taken over as maintainer, the app is actively updated, contrary to a review claiming it's "Almost abandoned" with a 1-star rating. This is misleading and incorrect.

Another review complains about "Misleading links and ownership," suggesting suspicions due to a hard fork. This is based on outdated information. All links have been updated to reflect the new GitHub repository I manage.

Outdated reviews are damaging the app's reputation and could mislead users. Here's what needs to happen:

1. We need a way to flag and remove reviews that are inaccurate or not relevant anymore.
2. The mentioned reviews need to be removed from the DuplicateFinder app page please.

These changes are critical for maintaining the app's integrity and helping users trust the app store.

André Théo LAURET
[Maintainer of DuplicateFinder]

**Comment by Fred-Zweig** [comment]
eldertek is right. Reviews that are clearly inaccurate and ill-intentioned should be first reported by the app's owner and then removed by an admin. Therefore, a 'Report' button with a text field for entering the motivation of reporting, should show up under each review when the app's owner is logged in, so that he can report the abusive review which unfairly damages the reputation of the app. Then an admin should verify the reported reivew and remove it.

The 'Anonymous' user who said 11 months ago that the Duplicate Finder was 'almost abandoned', who was proven wrong by Jos Poortvliet, who is a member of the Nextcloud team, also posted a huge number of bad reviews, around the same date (11 months ago), to almost all the apps in the app store, probably to make his own app rank higher. Apart from a few 1 sentence bad reviews like 'Almost abandoned', he left a lot of reviews with the following structure:
Development (commit history), Issues, Project management, Documentation, Localization, Releases (frequency). By doing so he wanted to induce the idea that his reviews are justified, because they are based on certain criteria: if the app is frequenly updated, if it has more solved than unsolved issues, if it has 'project management', etc. Although they seem legit, all these reviews are abusive. They clearly show that the respective 'Anonymous' user hasn't actually used the apps. He used to post many such bad reviews every single day, and it's clear he didn't have the time to install and test all the apps in the store in such a short interval. Before posting a review, a user should make at least the effort to install the app and test it, so that he can speak about the intrinsic qualities of the app: it does what is supposed to do, it has enough features, it's easy to use, etc. He sholdn't post a review based just on the frequency of updates, the number of solved/unsolved issues, the existence of 'project management', etc., whch don't say anything about the intrinsic qualities of the app. And even if he speaks only about the exterior qualities of the app, the user must at least install and test the app before leaving a review.

**Comment by eldertek** [comment]
How to delete these comments?

**Comment by bigcat88** [comment]
we will consider options on how to correctly implement the deletion option, I’ll get back with an answer in a couple of days

**Comment by bigcat88** [comment]
Please try to mark some spam comments for your apps, and we will see is this feature works or not.
Happy New Year, guys!

**Comment by eldertek** [comment]
Very good gift for 2024 ! Thanks !

**Comment by Fred-Zweig** [comment]
@bigcat88 Thank you very much for adding this feature. Now, with the 5 stars rating instead of Good, OK and Bad and with the 'Appeal for spam' button, the appstore looks very professional. If you could set the appstore to show the exact number of stars given up to the present that would be wonderful. As it is now, the number of stars displayed reflects the reality only after more than 5 reviews have been added. So, if an app has 4 reviews with 5 stars, the "Community rating" will show only 3 stars. The best way of showing the stars, used by WordPress plugins, Amazon products, etc. is to show the average star number up to the present moment and to specify the number of reviews from which the average was calculated. This means if an app has no reviews, the "Community rating" should display "No reviews yet" (and no stars). If the app has 1 review with 5 stars, the "Community rating" should display ' ***** from 1 review '. If it has 2 reviews of 3 stars it should display ' *** from 2 reviews ', etc.

**Comment by Fred-Zweig** [comment]
I added a new issue about the way the community rating stars should be displayed. I think this thread can be considered solved. Thank you @bigcat88 and Happy New Year.
